### PR TITLE
copy update

### PR DESCRIPTION
**Pull request by cokes518**

### CODE REVIEW

File: components/Hotspots/Checklist/Checklist.js [hunk 1]
```diff
@@ -86,15 +86,15 @@ const HotspotChecklist = ({
         },
         {
           sortOrder: 4,
-          title: 'Witness List',
+          title: 'Witnesses',
           detailText:
             witnesses.length > 0
               ? `Hotspot has ${witnesses.length} Hotspot${
```
Comment (danielcolinjames) [inline components/Hotspots/Checklist/Checklist.js:92]: like this, I believe it'll read: `Hotspot has 12 Hotspots.`

should it say something like: `Hotspot has been witnessed by 12 Hotspots` instead?

**Review by danielcolinjames — CHANGES_REQUESTED** [review]

File: components/Hotspots/Checklist/Checklist.js (hunk 1 above)
Comment (cokes518) [reply components/Hotspots/Checklist/Checklist.js:92]: ha oops! yes. that makes more sense.

File: components/Hotspots/Checklist/Checklist.js [hunk 2]
```diff
@@ -109,7 +109,7 @@ const HotspotChecklist = ({
                 } ago.`
               : `Hotspot hasn’t participated in a challenge yet.`,
           infoTooltipText:
-            'Participation in a challenge depends on having a witness list. Use the checkbox to see Hotspots in your list. It can take a few hours for challenges to include this Hotspot once a witness list is built.',
+            'Participation in a challenge depends on having a witnesses. Use the checkbox to see Hotspots in your list. It can take a few hours for challenges to include this Hotspot once a witness list is built.',
```
Comment (danielcolinjames) [inline components/Hotspots/Checklist/Checklist.js:112]: sorry noticed one more thing, "a witnesses" should be "witnesses"

**Review by danielcolinjames — APPROVED** [review]
🚢

**Pull request merged**